src/main/fc/fc_core.c

```diff
@@ -952,10 +943,7 @@ void taskMainPidLoop(timeUs_t currentTimeUs)
     gyroUpdate(currentTimeUs);
     DEBUG_SET(DEBUG_PIDLOOP, 0, micros() - currentTimeUs);
 
-    if (pidUpdateCountdown) {
-        pidUpdateCountdown--;
-    } else {
-        pidUpdateCountdown = setPidUpdateCountDown();
+    if (pidCounter++ % pidConfig()->pid_process_denom == 0) {
```

Comment:
What happens when `pidCounter` rolls over? We should probably reset it to `0` when the PID loop is executed.

Reply:
Rollover is well-defined modulo `(MAX_UINT8+1)`

Reply:
Understood!